Makefile

```diff
@@ -782,3 +786,23 @@ ifdef MLRUN_DOCKER_CACHE_FROM_TAG
 	done;
     MLRUN_DOCKER_CACHE_FROM_FLAG := $(MLRUN_BASE_IMAGE_DOCKER_CACHE_FROM_FLAG)
 endif
+
+.PHONY: verify-uv-version
+verify-uv-version:
+	@{ \
+	uv_version=$$(uv version | cut -d' ' -f2); \
+	result=$$(pysemver compare $$uv_version $(MLRUN_UV_VERSION)); \
+	if [ "$$result" -eq -1 ]; then \
+	  echo "Error: The running uv version ($$uv_version) is outdated. Upgrade uv to version $(MLRUN_UV_VERSION)."; \
+	  exit 1; \
+	fi; \
+	}
+
+.PHONY: update-mlrun-api-deps
+update-api-deps: verify-uv-version ## Update mlrun-api locked requirements file
```

Comment:
The .PHONY name doesn't match the rule name.
Anyway, maybe call it more explicitly `update-mlrun-api-deps-lock` (just to have 'lock' in the name to make it more clear what this does)